### ISSUE TITLE
Email feedback upon creation of ticket

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#10]
## User story
1. As a user that created a ticket
2. I want / need an email notification that tells me that my ticket was created successfully and some basic information about this ticket. (something like: Title, Ticket-Number, Description and Link to the Ticket)
3. So that I (again) get notified that my ticket was created and that I can check if the ticket was filled with correct/accurate information. And that I have prove that I can show, if necessary.

## Acceptance criteria
- [ ] The email was send to the right address
- [ ] All information inside the email is correct
- [ ] The time between creation and notification is short.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part
* The ticket must exist

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
